internal/parser/planparserv2/parser_visitor.go

```diff
@@ -1370,3 +1370,213 @@ func (v *ParserVisitor) VisitArrayLength(ctx *parser.ArrayLengthContext) interfa
 		nodeDependent: true,
 	}
 }
+
+func (v *ParserVisitor) VisitGeospatialEuqals(ctx *parser.GeospatialEuqalsContext) interface{} {
+	columnInfo, err := v.getChildColumnInfo(ctx.Identifier(), ctx.StringLiteral())
```

Comment:
no need to pass parameter `ctx.StringLiteral`

Reply:
refer 

childExpr, err := v.translateIdentifier(identifier.GetText())
		if err != nil {
			return nil, err
		}
		return toColumnInfo(childExpr), nil